home/tests/test_content_import_export.py

```diff
@@ -1262,6 +1262,51 @@ def test_import_ordered_content_sets_incorrect_locale_error(
 
         assert e.value.message == ["Locale pt does not exist."]
 
+    def test_import_ordered_content_sets_duplicate_name(
+        self, csv_impexp: ImportExport
+    ) -> None:
+        """
+        Importing CSV for ordered content sets with a duplicate name but unique slugs
+        should not throw an error.
+        """
+        csv_impexp.import_file("contentpage_required_fields.csv")
+
+        content = csv_impexp.read_bytes("ordered_content_same_name.csv")
+        csv_impexp.import_ordered_sets(content)
+
+        ordered_sets = OrderedContentSet.objects.all()
+        assert len(ordered_sets) == 2
+
+    def test_import_ordered_content_sets_duplicate_slug(
+        self, csv_impexp: ImportExport
+    ) -> None:
+        """
+        Importing CSV for ordered content sets with a duplicate name but unique slugs
+        should not throw an error.
```

Comment:
Yes, I would expect the last one in the file to take effect because we don't do a full file scan up front to see if there are duplicates in the file.

If we only had non-unique names and 2 sets with the same name then it's anyone's guess which one would be updated.

I'll update the comment.